fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -359,6 +360,7 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
                         safeInvalidateBrowserToolbarView()
                     },
                     onShowTranslationsDialog = {
+                        Events.translationsAction.record(Events.TranslationsActionExtra("main_flow_toolbar"))
```

Comment:
Opted to record the event at the entry points instead of the containing function as it felt more readable and precise for the corresponding Nimbs flag to me. LMWYT or what is typical!

Reply:
We typically add this to the resulting controller function since we can add a test for this that way. I think this is still the way I would suggest going about this.

Reply:
Makes sense, ty! Added a test where there was a controller and moved the rest to where the function was defined.